kong/plugins/influxdb-log/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
```

Comment:
Also here, re-use the data structure from the http-log plugin, instead of duplicating it.